applications/RomApplication/python_scripts/new_python_solvers_wrapper_rom.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import importlib
+
+import KratosMultiphysics
+from KratosMultiphysics import kratos_utilities
+from KratosMultiphysics.RomApplication import rom_solver
+
+def CreateSolverByParameters(model, solver_settings, parallelism, analysis_stage_module_name):
+
+    if (type(model) != KratosMultiphysics.Model):
+        raise Exception("input is expected to be provided as a Kratos Model object")
+
+    if (type(solver_settings) != KratosMultiphysics.Parameters):
+        raise Exception("input is expected to be provided as a Kratos Parameters object")
```

Comment:
Just suggesting, the more pythonic way is to use `isinstance` ryt?

Reply:
Agree. I copied and pasted. Updating it. Thanks.